src/dbwrapper.cpp

```diff
@@ -63,7 +63,7 @@ class CBitcoinLevelDBLogger : public leveldb::Logger {
 
                 assert(p <= limit);
                 base[std::min(bufsize - 1, (int)(p - base))] = '\0';
-                LogPrintf("leveldb: %s", base);
+                LogPrintf("leveldb: %s", base);  /* Continued */
```

Comment:
Since you are changing those lines anyway, why not use `LogPrintfContinuation` or something?

Reply:
My suggestion here: https://github.com/bitcoin/bitcoin/pull/12887#issuecomment-378936648 has inverted semantics. This `Continued` here comment indicates that there is a continuation log to follow. `LogPrintfContinuation()` would indicate that the log continues a previous log.

I think `LogPrintfContinuation()` could be useful, but it's a rebase-the-world PR since it removes the `\n` from every call to `LogPrintf()`

Reply:
Yes, please don't change every single log message. This solution is fine.